main.go

```diff
@@ -114,12 +114,13 @@ func addLogsHandler(mux *http.ServeMux, logs *supervisord.Logs) {
 }
 
 type gRPCServerDeps struct {
-	db             *reform.DB
-	prometheus     *prometheus.Service
-	server         *server.Server
-	agentsRegistry *agents.Registry
-	grafanaClient  *grafana.Client
-	checksService  *checks.Service
+	db                  *reform.DB
+	prometheus          *prometheus.Service
+	server              *server.Server
+	agentsRegistry      *agents.Registry
+	grafanaClient       *grafana.Client
+	checksService       *checks.Service
+	dbaasControllerConn *dbaas.Client
```

Comment:
Just a comment: I would name is dbassClient and not ControllerConn. 
Only to make it more clear

Reply:
Well, I agree that is it not the best name.
But it is still a connection to the *Controller* of DBaaS, but not directly to DBaaS.
So, I hesitate here about renaming.

Reply:
But struct is dbaas.Client, right?

Reply:
dbaasControllerClient?